### ISSUE TITLE
Cannot unit test components using vue-tree with TypeScript

### ISSUE THREAD

**Issue by dandee** [grapoza/vue-tree#232]
**Describe the bug**
Cannot unit test components using vue-tree with TypeScript.

**To Reproduce**
1. Checkout https://github.com/dandee/vue-tree-repro repository
2. Run `npm install; npm test`
3. See error:

```
 FAIL  tests/unit/items.spec.ts
  ● Test suite failed to run

    /Users/goofy/Development/sandbox/vue-tree-repro/node_modules/@grapoza/vue-tree/index.js:5
    import TreeView from './components/TreeView.vue';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      2 | import { setActivePinia, createPinia, PiniaPlugin } from 'pinia';
      3 | import { shallowMount, createLocalVue } from '@vue/test-utils';
    > 4 | import TreeView from '@grapoza/vue-tree'
        | ^
      5 |
      6 | import { itemsStore } from '@/store/items';
      7 | import Items from '@/components/items.vue';

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)
      at Object.<anonymous> (tests/unit/items.spec.ts:4:1)
```

**Expected behavior**
No error while unit testing.

**Screenshots**

 - Browser NodeJS
 - Version 16.11.1
 
<img width="1069" alt="Screenshot 2021-10-18 at 21 45 51" src="https://user-images.githubusercontent.com/890495/137797844-c5390ad9-ad2f-4536-bd9f-6569a4a29636.png">

**Additional context**
It looks like 'index.js' includes './components/TreeView.vue' instead of './src/components/TreeView.vue' maybe? But it also may be a red herring.

**Comment by dandee** [comment]
Hey, do you think you'd have time to address this issue?

**Comment by grapoza** [comment]
Hey, sorry about this one. I had it flagged for follow up and then my schedule got away from me. I have some time tonight to start looking at it, so I'll try to get something back to you ASAP.

**Comment by grapoza** [comment]
Well now I feel even worse for taking so long to look at it. It does indeed look like a path issue, but not for index.js. The imports you're using should grab the main module export from the package, but _that_ had a bad path of `dist/vue-tree.umd.min.js` (`dist` is in the source, but not in the built npm package). I fixed that locally to try out your reproduction repo and it found the module without a problem. I also made a couple of tweaks to the test to get it to pass that I think are right, namely adding boostrap imports to resolve all the `b-*` tags and I changed the assertion to find `'tree-view-stub#tree` since it's shallow mounted, so I think it proves out the changes I'm making.

I have no idea how this has gone a year and a half without it getting found. It's probably a combination of people manually mocking the component and other people not reporting the issue, so thanks for taking the time to do so!

I'll work on getting a 3.0.5 out tonight with the updated package.json, let me know if that takes care of the issue for you.
